src/fastcs/datatypes.py

```diff
@@ -10,7 +10,16 @@
 import numpy as np
 from numpy.typing import DTypeLike
 
-T = TypeVar("T", int, float, bool, str, enum.Enum, np.ndarray)
+T = TypeVar(
+    "T",
+    int,  # fastCS.datatypes.Int
+    float,  # fastCS.datatypes.Float
+    bool,  # fastCS.datatypes.Bool
+    str,  # fastCS.datatypes.String
+    enum.Enum,  # fastCS.datatypes.Enum
+    np.ndarray,  # fastCS.datatypes.Waveform
+    list[tuple[str, DTypeLike]],  # fastCS.datatypes.Table
```

Comment:
Very picky. I don't think you need the whole import path here, but if you do then it needs to be `fastcs` in lower case

Reply:
Whoops, nah not too picky at all.